consensus/core/src/authority_node.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use parking_lot::RwLock;
```

Comment:
For some reason my editor automatically inserts imports to its group (std/external/crate).

Reply:
I'll look into it but didn't find a quick setting to support this. Until then I'll try to manually resove this